cg/meta/upload/scoutapi.py

```diff
@@ -142,22 +184,31 @@ def _include_files(self, data, hk_version, scout_hk_map, extra_tag=None, skip_mi
                     raise FileNotFoundError(f"missing file: {scout_key}")
 
     @staticmethod
-    def _is_family_case(data):
-        return any(sample['father'] or sample['mother'] for sample in data['samples'])
+    def _is_family_case(data: dict):
+        """Check if there are any linked individuals in a case"""
+        for sample in data["samples"]:
+            if sample["mother"] and sample["mother"] != "0":
```

Comment:
```suggestion
            if sample["mother"] != "0":
```
should give same result, right?

Reply:
If for some reason `sample['mother'] = None` your suggestion would not work. I did this test since this is the old behaviour and am not sure if it could show up again

Reply:
That's right, lets move on ;)